pystiche_papers/ulyanov_et_al_2016/_loss.py

```diff
@@ -56,7 +91,17 @@ def calculate_score(
         # uses reduction="mean" which also includes the batch_size. However, the
         # score is divided once more by the batch_size in the reference implementation.
         batch_size = image.extract_batch_size(input_repr)
-        return score / batch_size
+        # https://github.com/pmeier/texture_nets/blob/b2097eccaec699039038970b191780f97c238816/src/content_loss.lua#L23-L25
```

Comment:
[L162](https://github.com/pmeier/texture_nets/blob/b2097eccaec699039038970b191780f97c238816/stylization_train.lua#L162) seems to only divide twice by the batch size for logging. In [L165](https://github.com/pmeier/texture_nets/blob/b2097eccaec699039038970b191780f97c238816/stylization_train.lua#L165) you can see that the original loss is returned.

Following this I double checked with the reference implementation. IMO, the content loss behaves the same as a default `ops.FeatureReconstructionOperator`. If you are in doubt or have a test case where you think this is not the case, please let me know. Otherwise, I think we can remove this custom implementation completely.

Reply:
I think it would be a good idea to split this into separate PR.